book/makefile

```diff
@@ -136,15 +137,11 @@ low-res-book-epub/OEBPS/%.jpg: %.jpg
 	mkdir -p $(dir $@)
 	$(CONVERT_PIC) $< $(REDUCE_PIC_COLOR) $@
 
-low-res-book-epub/OEBPS/%.png: %.png
-	mkdir -p $(dir $@)
-	$(CONVERT_PIC) $< $(REDUCE_PIC_COLOR) $@
-
 epub/bw_book.epub: copy_ebook_files $(bw_images)
 	cd bw-book-epub; zip -q0X ../epub/bw_book.epub mimetype
 	cd bw-book-epub; zip -q9XrD ../epub/bw_book.epub ./
 
-epub/low_res_book.epub: copy_ebook_files_cl $(low_res_images)
+epub/low_res_book.epub: copy_ebook_files_low_res $(low_res_images)
 	cd low-res-book-epub; zip -q0X ../epub/low_res_book.epub mimetype
 	cd low-res-book-epub; zip -q9XrD ../epub/low_res_book.epub ./
```

Comment:
I was just thinking if I could find a more generic solution for the test, it could be reused (it's not generic right now though). It can easily be added here if you think that is better.

Reply:
Fair point we could have a generic check size thing for the B&W and others... not sure we will ever need it in practice.

Do whatever you think is best (but put the rule around the spell-check and check-tex if you decide to go that way).

Thanks.

Reply:
I took out the separate rule. If we actually need it, we can make that change later